### ISSUE TITLE
Unable to run `energy_central` in sample project

### ISSUE THREAD

**Issue by willu47** [nismod/smif#355]
Unable to run `energy_central`?

I'm getting `AttributeError: 'EnergyAgent' object has no attribute 'interventions'`

Reported by @fcooper8472

**Comment by willu47** [comment]
I am unable to replicate this.

**Comment by fcooper8472** [comment]
Yep - I just wiped my sample project and recreated it and no longer have that issue.

Assume it was spurious... happy to close this if you are?
